### PR TITLE
Update Twilio Package.md and Module.md files

### PR DESCRIPTION
**Pull request by LahiruRajapaksha**
Related https://github.com/wso2-enterprise/choreo/issues/11866
md1
md2

### CODE REVIEW

**Review by HasithaAthukorala — APPROVED** [review]

File: asyncapi/twilio/Module.md [hunk 1]
```diff
@@ -1,31 +1,34 @@
 ## Overview
 
 The Twilio Trigger allows you to listen to Twilio SMS and Call status change events.
-1. Listen to incoming message events and message status change callback events from the twilio SMS.
-2. Listen to incoming call events and call status change callback events from the twilio Voice Call.
+1. Listen to incoming message events and message status change callback events from the Twilio SMS.
+2. Listen to incoming call events and call status change callback events from the Twilio Voice Call.
```
Comment (Nashaath) [inline asyncapi/twilio/Module.md:5]: This doesn't need to be an ordered list. So shall we make these bullet points? 
Suggestion:
The Twilio trigger allows you to listen to Twilio SMS and call status change events similar to the following:
- Incoming message events and message status change callback events from Twilio SMS.
- Incoming call events and call status change callback events from Twilio voice call.

File: asyncapi/twilio/Module.md [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
 
 This module supports [Twilio Basic API 2010-04-01](https://www.twilio.com/docs/all) version.
```
Comment (Nashaath) [inline asyncapi/twilio/Module.md:7]:
```suggestion
This module supports the [Twilio Basic API 2010-04-01](https://www.twilio.com/docs/all) version.
```

File: asyncapi/twilio/Module.md [hunk 3]
```diff
@@ -1,31 +1,34 @@
...
 
 ## Prerequisites
 Before using this trigger in your Ballerina application, complete the following:
 
-* Sign up to twilio and create a Twilio account (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
+* Sign up to Twilio and create a Twilio account (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
```
Comment (Nashaath) [inline asyncapi/twilio/Module.md:12]:
```suggestion
* Sign up to Twilio and create a Twilio account. For step-by-step instructions, see [View and Create New Accounts in Twilio Console] (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
```

File: asyncapi/twilio/Module.md [hunk 4]
```diff
@@ -1,31 +1,34 @@
...
 * Get a twilio phone number which needs to purchase through Twilio to send messages or make phone calls using Twilio.
```
Comment (Nashaath) [inline asyncapi/twilio/Module.md:13]:
```suggestion
* Follow the steps below to purchase a Twilio phone number to send messages and make phone calls using Twilio:
```

File: asyncapi/twilio/Module.md [hunk 5]
```diff
@@ -1,31 +1,34 @@
...
     1. Navigate to the Phone Numbers page in your console.
```
Comment (Nashaath) [inline asyncapi/twilio/Module.md:14]: Is `Phone Numbers` page correct. [1] says it is `Buy a Number` page. So do we need to change?
Suggestion if we are to change:
"Navigate to the **Buy a Number** page in the Twilio console."

[1] https://support.twilio.com/hc/en-us/articles/223135247-How-to-Search-for-and-Buy-a-Twilio-Phone-Number-from-Console

Comment (LahiruRajapaksha) [reply asyncapi/twilio/Module.md]: When login to the console there is a section called `Phone Numbers` in the left pannel

File: asyncapi/twilio/Module.md [hunk 6]
```diff
@@ -1,53 +1,59 @@
...
 
-* Sign up to twilio and create a Twilio account (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
-* Get a twilio phone number which needs to purchase through Twilio to send messages or make phone calls using Twilio.
-    1. Navigate to the Phone Numbers page in your console.
-    2. Click Buy a Number to purchase your first Twilio number.
+* Sign up to Twilio and create a Twilio account.For step-by-step instructions, see [View and Create New Accounts in Twilio Console] (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
+* Follow the steps below to purchase a Twilio phone number to send messages and make phone calls using Twilio:
+    1. Click on the **Phone Numbers -> Manage** at your left navigation panel **Develop** view.
+    2. Click **Buy a Number** to purchase your first Twilio number.
 * Register the request URL
```
Comment (Nashaath) [inline asyncapi/twilio/Module.md:16]:
```suggestion
* Follow the steps below to register the request URL:
```

File: asyncapi/twilio/Module.md [hunk 7]
```diff
@@ -1,53 +1,59 @@
...
 ## Prerequisites
 Before using this trigger in your Ballerina application, complete the following:
 
-* Sign up to twilio and create a Twilio account (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
-* Get a twilio phone number which needs to purchase through Twilio to send messages or make phone calls using Twilio.
-    1. Navigate to the Phone Numbers page in your console.
-    2. Click Buy a Number to purchase your first Twilio number.
+* Sign up to Twilio and create a Twilio account.For step-by-step instructions, see [View and Create New Accounts in Twilio Console] (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
+* Follow the steps below to purchase a Twilio phone number to send messages and make phone calls using Twilio:
+    1. Click on the **Phone Numbers -> Manage** at your left navigation panel **Develop** view.
```
[the remark this answers is not in the document]
Comment (LahiruRajapaksha) [reply asyncapi/twilio/Module.md:14]: In here do we have to remove the bold of the word `Develop`?

**Review by Nashaath — APPROVED** [review]

**Pull request merged**